include/picongpu/plugins/ResourceLog.hpp

```diff
@@ -103,67 +90,39 @@ namespace picongpu
 
         void notify(uint32_t currentStep)
         {
-            //
-            // Create property tree which contains the resource information
-            using boost::property_tree::ptree;
-            ptree pt;
+            std::map<std::string, size_t> valueMap;
 
             if(contains(propertyMap, "rank"))
-            {
-                size_t rank = static_cast<size_t>(Environment<simDim>::get().GridController().getGlobalRank());
-                pt.put("resourceLog.rank", rank);
-            }
+                valueMap["resourceLog.rank"] = static_cast<size_t>(Environment<simDim>::get().GridController().getGlobalRank());
 
             if(contains(propertyMap,"position"))
             {
-                DataSpace<simDim> currentPosition = Environment<simDim>::get().GridController().getPosition();
-                pt.put("resourceLog.position.x", currentPosition[0]);
-                pt.put("resourceLog.position.y", currentPosition[1]);
-                pt.put("resourceLog.position.z", currentPosition[2]);
+                auto const currentPosition = Environment<simDim>::get().GridController().getPosition();
+                char coord = 'x';
```

Comment:
(Not important) I find an array of 'x', 'y', 'z' a little cleaner than the character arithmetic inside a loop, smth like that:
```cpp
char const axisName[] = { 'x', 'y', 'z' };
for( size_t d = 0; d < simDim; ++d )
    valueMap[std::string("resourceLog.position.") + axisName[d]] = currentPosition[d];
```

Reply:
Great idea, I love explicitly named things :)